torch/nn/quantized/modules/channelshuffle.py

```diff
@@ -0,0 +1,16 @@
+import torch
+
+class ChannelShuffle(torch.nn.ChannelShuffle):
```

Comment:
if the float version works for quantized inputs as well, I don't think we will need to define an extra quantized module

Reply:
it should be a part of this list in that case I think: https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/fx/_lower_to_native_backend.py#L122

Reply:
OK. I have removed them.